src/front/app/components/globals/Dropdown.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Manager, Reference, Popper } from 'react-popper';
+import Button from './Button';
+
+const popperPlacements = [
+  'auto',
+  'top',
+  'bottom',
+  'left',
+  'right',
+  'auto-start',
+  'top-start',
+  'bottom-start',
+  'left-start',
+  'right-start',
+  'auto-end',
+  'top-end',
+  'bottom-end',
+  'left-end',
+  'right-end',
+];
+
+class Dropdown extends Component {
+  static propTypes = {
+    toggler: PropTypes.func,
+    activePlacement: PropTypes.oneOf(popperPlacements),
+    className: PropTypes.string,
+    children: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.node,
+      PropTypes.instanceOf(Component),
+    ]).isRequired,
+  };
+
+  static defaultProps = {
+    activePlacement: 'bottom',
+    toggler: ({
+      ref,
+      isOpen,
+      onClick,
+    }) => {
+      const btnGroupClasses = classNames({
+        'btn-group': true,
+        dropdown: true,
+        show: isOpen,
+      });
+
+      const btnClasses = classNames({
+        'dropdown-toggle': true,
+      });
+
+      return (
+        <div className={btnGroupClasses}>
+          <Button
+            type="primary"
+            className={btnClasses}
+            inputRef={ref}
+            onClick={onClick}
```

Comment:
You could call the method `openDropdown`. Not bad as onClick, but would be a bit more clear if you call it openDropdown.